src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -240,6 +240,30 @@ public void v3(string ns, string attrNs, string expectedNs)
             CompareWildcardNamespaces(expectedNs, attributeWildcard.Namespace);
         }
 
+        [Fact]
+        //[Variation(Desc = "new instance return null namespace (issue #41704")]
+        public void v4()
+        {
+            var any = new XmlSchemaAnyAttribute();
+            Assert.Null(any.Namespace);
+            return;
+        }
+
+        [Fact]
+        //[Variation(Desc = "without namespace returns null (issue #41704"")]
+        public void v5()
+        {
+            var xsd = @"<xs:schema xmlns:xs='http://www.w3.org/2001/XMLSchema' targetNamespace='ns'>
+                        <xs:complexType name = 't'>
+                            <xs:anyAttribute/>
+                        </xs:complexType>
+                        </xs:schema>";
+            var xs = XmlSchema.Read(new StringReader(xsd), null);
+            var any = ((XmlSchemaComplexType)xs.Items[0]).AnyAttribute;
```

Comment:
Our style only allows `var` when the type is "obvious", which means a `new`, `(cast)`, or `as cast`.  All three of the `var`s in this method should be replaced with the actual variable types.